data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3675 @@
+				die
+			label one
+			action
+				outfit "Gold Bars" 1
+			``
+				goto dropbars
+			label three
+			action
+				outfit "Gold Bars" 3
+			``
+				goto dropbars
+			label five
+			action
+				outfit "Gold Bars" 5
+			label dropbars
+			`	You drop the bars and then come out of the ship without too much worry, as you do not carry any items on you. Afterwards you go around the ship to the location where they are partially buried in the sand, and take them discreetly with some members of your crew. You have to silence one or two curious pirates but that is about it, and you can now head back to your ship. As expected they have noticed the theft, as you notice them launching before you can do so yourself. This will be a bumpy ride.`
+				flee
+			
+			label caught
+			`	You do not manage to find the item here, however you have attracted way too much suspicions by looking and opening crates everywhere you could. One local worker asks you something about mechanics. Being no engineer yourself, you cannot make any sense of what he says, even less formulate a correct response, and he denounces you. You are captured and sold as a slave.`
+				die
+			label found
+			choice
+				`	(Take an alien looking modified torpedo warhead.)`
+					goto stupid
+				`	(Take an alien skull.)`
+					goto wrongitem
+				`	(Take a weird looking small cube.)`
+			`	Surprisingly, you appear to have found what you were looking for: a small cube, completely regular and seemingly alien in design. Maybe the warlord was not aware of its value, considering she placed it here, without much protection, and without even trying to hide it. This has to be the item that He Yi wants, so you take it with you. Its small size makes it easy to conceal as you make your way off the ship and back to your own. This probably will not go unnoticed for long however, so you launch as soon as you can. The original pirate fleet is already heading your way, smelling something suspicious...`
+				launch
+			label stupid
+			`	You try to disassemble the torpedo, but you do not have enough time to do so.`
+				goto caught
+
+			label bargain
+			`	You ask around and find the first mate of the warlord, Mip. He seems intrigued by your request for a small and precious item that their boss has just acquired. "I think I know of which item you speak, indeed the old warlord was aware of it and orderer it delivered to him, something Kira knows not of yet... She does not need to. Indeed that item is very mysterious, which sells well on the black market. Also I do not know you, so you would have to give quite a large sum if you really want it, especially because you cannot just go and speak to my boss as if you knew her." He thinks a bit, looking at what kind of things you are wearing, estimating how much you would be willing to pay, he also looks on his pad, probably for records of your ship and fleet. "10 million credits," he says in a definitive tone.`
```

Comment:
```suggestion
			`	You ask around and find the first mate, Mip. He seems intrigued by your request for a small and precious item that their boss has just acquired. "Yeah, I think I know what you're talking about - matter o' fact, the old boss knew about it and ordered it delivered to him. Kira doesn't know about it yet - I guess she doesn't need to know. That item is a bit mysterious, which adds some value on the black market. Also, 'cause I don't know you, you'd have to pony up some serious moolah if you really want it, especially since you can't just go and talk to the boss about it." He thinks a bit, assessing your appearance and clothing to estimate how much you might be willing to pay. He also checks his pad, probably searching for records of your ship and fleet. "10 million credits," he says in a definitive tone.`
```